core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy

```diff
@@ -8,12 +8,6 @@ def l=namespace(lib.LayoutTagLib)
 def st=namespace("jelly:stapler")
 
 l.layout(permission:app.SYSTEM_READ, title:my.displayName) {
-    l.side_panel {
-        l.tasks {
-            l.task(icon:"icon-up icon-md", href:rootURL+'/', title:_("Back to Dashboard"))
-            l.task(icon:"symbol-settings", href:"${rootURL}/manage", title:_("Manage Jenkins"))
-        }
-    }
```

Comment:
Time to apply the new config form to this page, so the sidepanel doesn't look so empty 😁